src/components/ImmutableDevices/ImmutableDevices.js

```diff
@@ -40,10 +38,6 @@ const ImmutableDevices = ({
     return [...mergeAppColumns(filteredColumns), ...edgeColumns];
   };
 
-  const onRowClick = (_key, systemId) => {
-    navigate(`/insights/inventory/${systemId}?appName=vulnerabilities`);
```

Comment:
This became obsolete with using the `Link` in the `TitleColumn` and the react router oddnesses being resolved, not requiring to pass in a navigate ensuring it is referencing the correct router to navigate.